lib/absinthe/pipeline/error_result.ex

```diff
@@ -1,20 +0,0 @@
-defmodule Absinthe.Pipeline.ErrorResult do
```

Comment:
I wonder if some users are in fact using this even if Absinthe internals aren't using it... It was part of the docs

Reply:
Happy to leave this in, but easy enough for users to put back if they depend on it.

Reply:
Let's move forward and address it if anyone reports any issues